tasks/us/bls.py

```diff
@@ -265,23 +273,9 @@ def populate(self):
 class AllQCEW(WrapperTask):
 
     def requires(self):
-        for year in range(2012, 2016):
+        maxyear, maxqtr = [int(n) for n in MAX_TIMESPAN.split('Q')]
+        for year in range(2012, maxyear + 1):
             for qtr in range(1, 5):
+                if year == maxyear and qtr > maxqtr:
```

Comment:
I would do the opposite condition:
```
if !(year == maxyear and qtr > maxqtr):
 yield QCEW(year=year, qtr=qtr)
```

Reply:
I prefer applying some Bool algebra and ellaborate it a bit:
```
if year < maxyear or qtr <= maxqtr:
    yield QCEW(year=year, qtr=qtr)
```